client/src/assets/images/map/markers/index.js

```diff
@@ -0,0 +1,18 @@
+import crosshairs from './crosshairs.png';
+import lhq from './lhq.png';
+import pin from './pin.png';
+
+const mapMarker = (name) => {
```

Comment:
Why is this component in assets/images? Could it not be a stateless component in /components ?

Reply:
I wasnt sure on the normal for something like this. react wont let you dynamic load images, so this code is kinda an asset wrapper so I thought it could like with the assets it loads?

Reply:
Should be OK - we may revisit - I typically keep all the stateless components together - let's proceed so we can keep things moving.